### DISCUSSION TITLE
Improvements to Outline hierarchy for .R files

### DISCUSSION THREAD

**Discussion by brndngrhm** [posit-dev/positron#4452] [Feedback] [answered]
I was wondering if Positron can add better organization in the Outline pane.

In VSCode, the Outline pane does a good job of creating a nested hierarchy that matches the code comment sections when 4 dashes are used (e.g. `# some coment ----`) . Using this, users can nicely organize the outline pane like so:

![image](https://github.com/user-attachments/assets/e5f21077-3518-49a2-b326-d67efda884e1)

But the Outline pane for same .R file in Positron doesn't look as organized, but I do really like how the function parameters are displayed!

![image](https://github.com/user-attachments/assets/20c3c72d-41e1-476c-ac18-ec4f626d596d)

**Comment by juliasilge** [comment, marked as answer]
I believe we have this tracked in https://github.com/posit-dev/positron/issues/3822. Take a look and see if you have additional info to add! 🙌
